src/core/Akka/Event/DeadLetter.cs

```diff
@@ -104,7 +104,7 @@ public sealed class SuppressedDeadLetter : AllDeadLetters
         /// <exception cref="ArgumentNullException">
         /// This exception is thrown when either the sender or the recipient is undefined.
         /// </exception>
-        public SuppressedDeadLetter(IDeadLetterSuppression message, IActorRef sender, IActorRef recipient) : base(message, sender, recipient)
```

Comment:
Breaking API change here, kind of - needed to relax this type constraint otherwise we'd lose data in the `SuppressedDeadLetter` logging